packages/story-editor/src/components/panels/design/layer/layer.js

```diff
@@ -342,6 +342,7 @@ function Layer({ element }) {
               title={__('Duplicate Layer', 'web-stories')}
               hasTail
               isDelayed
+              placement={'bottom-end'}
```

Comment:
In addition to updating this placement, we should be importing `Tooltip` from `Story-Editor` to get the correct placements. This should fix the negative values issue as well.